cookbooks/aws-parallelcluster-slurm/recipes/config/login_nodes_daemon_service.rb

```diff
@@ -0,0 +1,54 @@
+require 'yaml'
+config = YAML.safe_load(File.read(node['cluster']['cluster_config_path']))
+GRACETIME_PERIOD = config["LoginNodes"]["Pools"][0]["GRACETIME_PERIOD"]
+
+# Create configuration file
+template "#{node['cluster']['shared_dir']}/login_nodes_daemon_config.json" do
+  source 'login_nodes_daemon_service/login_nodes_daemon_config.json.erb'
+  owner 'root'
+  group 'root'
+  mode '0644'
+  variables(
+    {
+      GRACETIME_PERIOD: GRACETIME_PERIOD,
+    }
+  )
+end
+
+# Create on_termination_script.sh
+template "#{node['cluster']['shared_dir']}/login_nodes_on_termination_script.sh" do
+  source 'login_nodes_daemon_service/login_nodes_on_termination_script.sh.erb'
+  owner 'root'
+  group 'root'
+  mode '0755'
+end
+
+# Create daemon_script.sh
+template "#{node['cluster']['shared_dir']}/login_nodes_daemon_script.sh" do
+  source 'login_nodes_daemon_service/login_nodes_daemon_script.sh.erb'
+  owner 'root'
+  group 'root'
+  mode '0755'
```

Comment:
Here and in every script defined in this PR that is meant to be executed by root.

`0755` is too permissive as it would grant execution access to everyone.
You should rather use `0744` so that root will have read-write-execute access and the others would have only read access.

Reply:
Have fixed it! Thank you.